Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.cpp

```diff
@@ -295,78 +369,111 @@ namespace AZ
                         DeAllocateImageTilesInternal(image, RHI::GetImageSubresourceIndex(mipIndex, arrayIndex, descriptor.m_mipLevels));
                     }
                 }
+                image.UpdateResidentTilesSizeInBytes(TileSizeInBytes);
             }
         }
-        
+
+        bool StreamingImagePool::ShouldUseTileHeap(const RHI::ImageDescriptor& imageDescriptor) const
+        {
+            if (m_enableTileResource)
+            {
+                // D3D12_RESOURCE_DIMENSION_TEXTURE1D is not supported for tier 1 tile image resource
+                if (imageDescriptor.m_dimension == RHI::ImageDimension::Image1D)
+                {
+                    return false;
+                }
+
+                /**
+                * DirectX has weird limitation in the implementation where texture arrays are not currently supported
+                * (despite the spec saying they are):
+                *
+                * 'On a device with Tier 2 Tiled Resources support, Tiled Resources cannot be created with both more
+                *  than one array slice and any mipmap that has a dimension less than a tile in extent.
+                *  Hardware support in this area was not able to be standardized in time to be included in D3D'
+                *
+                * Need to follow up to figure out if this is a permanent limitation or whether it will be lifted in an
```

Comment:
nit: probably worth opening a GHI to track this.

Reply:
I update the code with accurate condition check